applications/crossbar/doc/token_auth.md

```diff
@@ -65,4 +76,14 @@ URL segment: `/token_auth`
 
 If you'd like to invalidate an authentication token programmatically (versus letting the system expire the token), you can issue a `DELETE`:
 
-    curl -v -X DELETE -H "X-Auth-Token: {AUTH_TOKEN}" http://server.com:8000/v1/token_auth
+```shell
+curl -v -X DELETE -H "X-Auth-Token: {AUTH_TOKEN}" http://server.com:8000/v1/token_auth -d '{"data": {}}'
+```
+
+```json
```

Comment:
I'm on the fence about this separation. I guess it shouldn't matter anymore since Slate is out so...let's see how it goes.